app/assets/stylesheets/base.css.scss

```diff
@@ -348,7 +292,3 @@ fieldset.form-group {
 .btn.red-border {
   border-color: #660000;
 }
-
-#logo {
-  color: #406e8e;
-}
```

Comment:
Removing the color is fine for me. However, we should also update the following files:

- public/apple-touch-icon.png
- public/apple-touch-icon-precomposed.png
- public/favicon.ico
- public/favicon.png

Reply:
On second thought, the color for the anchor logo is quite nice and gives a distinct color. But I am open for making it in the similar color to other parts. Therefore, I will put the logo color back.

Reply:
I don't want to influence you on that decision and think an overall design is desired here. I am just desiring to have a consistent appeal 😁.

Reply:
don't have any strong opinion to be honest :D